src/incipit/processing.py

```diff
@@ -176,6 +205,9 @@ def draw_regions_on_images(
         # defensively copy
         image_with_regions = image.copy()
 
+        # adjsut region positioning
```

Comment:
@Allan518 Typo. And be a bit more descriptive.

Reply:
@jlumbroso Corrected it, please review.